packages/exo/test/test-heap-classes.js

```diff
@@ -10,6 +10,19 @@ import {
 } from '../src/exo-makers.js';
 import { GET_INTERFACE_GUARD } from '../src/exo-tools.js';
 
+const NoExtraI = M.interface('NoExtra', {
+  foo: M.call().returns(),
+});
+
+test('what happens with extra arguments', t => {
+  const exo = makeExo('WithExtra', NoExtraI, {
+    foo(x) {
+      t.is(x, undefined);
+    },
+  });
+  exo.foo(8);
```

Comment:
takes a bit to know this value is irrelevant.
```suggestion
  exo.foo('an extra arg');
```

Reply:
Done